petergreve/application/test/stores/message-store.js

```diff
@@ -0,0 +1,35 @@
+var rewire = require("rewire");
+var Constants = require('../../app/constants/constants')
+var MessageStore = rewire('../../app/stores/message-store')
+import Dispatcher from '../../app/dispatcher/dispatcher'
+
+describe('Message Store methods', () => {
+	
+	before(() => {
+        MessageStore.__set__('_messages',[])

```

Comment:
Why is this an anti-pattern?. I'm just making sure the store is empty so I can test it independently . Yes it does work